accessibility-checker.php

```diff
@@ -134,9 +133,8 @@
 /**
  * Filters and Actions
  */
-add_action( 'admin_enqueue_scripts', 'edac_admin_enqueue_scripts' );
-add_action( 'admin_enqueue_scripts', 'edac_admin_enqueue_styles' );
-add_action( 'wp_enqueue_scripts', 'edac_enqueue_scripts' );
+add_action( 'admin_enqueue_scripts', array( 'EDAC\Admin\Enqueue_Admin', 'enqueue' ) );
```

Comment:
Shouldn't this be moved to the `class-admin.php` so it's never executed unless in the admin, consistency, and to keep the main plugin file clean?